Modules/posixmodule.c

```diff
@@ -14962,6 +14962,314 @@ os_waitstatus_to_exitcode_impl(PyObject *module, PyObject *status_obj)
 #endif
 
 
+#if defined(MS_WINDOWS)
+/*[clinic input]
+os._isdir
```

Comment:
For consistency, it occurs to me that these functions should follow the same naming scheme as `nt._path_normpath()` and `nt._path_splitroot()`. Also, it would be better to move them up to line 4492, just after `os__path_splitroot_impl()`, so that the `os.path` accelerators are grouped together.

You'll have to rerun clinic if you make these changes.

```suggestion
os._path_isdir
```